docs/modules/ROOT/pages/how-to-guides/Troubleshooting.adoc

```diff
@@ -0,0 +1,120 @@
+= Troubleshooting Application Deployment Issues on Payara Cloud
+
+Deploying a new application to Payara Cloud may occationally be accompanied by errors or issues that prevent the application from deploying and running smoothly. Below is a comprehensive guide to help you troubleshoot some of these common problems.
+
+[[no-deployed-application-artefact]]
+== No Deployed Application Artifact
+
+=== Symptoms
+You've initiated the deployment process, but there is no sign of your application artifact in the Payara Cloud Dashboard.
+
+=== Troubleshooting Steps
+1. **Check Upload Status**: Make sure the artifact has been successfully uploaded. Look for any error messages during the upload process.
+2. **Inspect Logs**: Check the deployment logs for any error messages that could indicate what went wrong.
+3. **Verify Artifact Type**: Ensure that you are uploading a supported artifact type (.war) created from a supported Jakarta EE API version (10, 8).
+
+[[application-not-reachable]]
+== DNS Propagation for Application URL Not Fully Done
+
+=== Symptoms
+Your application has been deployed, but navigating to the provided URL results in a DNS error.
+
+=== Troubleshooting Steps
+1. **Wait**: DNS propagation can take up to 48 hours (depending on your location). You may need to wait before the changes propagate across all DNS servers.
+2. **Check DNS Configuration**: Verify that your DNS settings are correctly configured in the Payara Cloud Dashboard.
+3. **Use DNS Propagation Check Tools**: Utilize online tools (eg. https://dnschecker.org) to check the status of DNS propagation globally.
+
+[[application-not-accessible-on-custom-domain]]
+== Custom Domain May Still Not Have Fully Propagated
+
+=== Symptoms
+You've configured a custom domain, but navigating to it doesn't resolve to your Payara Cloud application.
+
+=== Troubleshooting Steps
+1. **Wait**: Similar to DNS propagation, custom domains may also take time to fully propagate.
+2. **Verify Domain Configuration**: Double-check to ensure that your custom domain has been correctly set up in your Payara Cloud settings.
```

Comment:
I'm not 100% sure how to link the various docs. Here we should link to --> https://docs.payara.fish/cloud/docs/how-to-guides/How%20to%20Use%20Custom%20Domains.html

Reply:
`2. **Check DNS Configuration**: Verify that your DNS settings are correctly configured in the Payara Cloud Dashboard. Please refer to xref:how-to-guides/How to Use Custom Domains.adoc[How to Use Custom Domains]`

You can add this. It will link to the page for 'How to Use Custom Domains'